
2022-10-24 15:09:13.176 - [Debug] [Auth] Attacker 117.16.44.111 trying to authenticate with 'publickey'
2022-10-24 15:09:13.176 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-24 15:09:13.177 - [Debug] [Auth] Attacker: 117.16.44.111 has so far made 1 attempts. Remaining: 5 attempts
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:1240910) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.244',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-24 15:10:17.316 - [Info] MITM Version: 2
2022-10-24 15:10:17.317 - [Info] Auto Access Enabled: true
2022-10-24 15:10:17.317 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-24 15:10:17.325 - [Info] SSH man-in-the-middle server for 10.0.3.244 listening on 10.0.3.1:1003
2022-10-24 15:10:17.437 - [Debug] [Connection] Attacker connected: 69.236.215.6 | Client Identification: SSH-2.0-Go
2022-10-24 15:10:17.493 - [Debug] [Connection] Attacker connected: 31.165.5.155 | Client Identification: SSH-2.0-Go
2022-10-24 15:10:17.683 - [Debug] [Connection] Attacker connected: 120.236.79.182 | Client Identification: SSH-2.0-Go
2022-10-24 15:10:17.693 - [Debug] [Auth] Attacker 69.236.215.6 trying to authenticate with 'none'
2022-10-24 15:10:17.693 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-24 15:10:17.792 - [Debug] [Auth] Attacker 69.236.215.6 trying to authenticate with 'publickey'
2022-10-24 15:10:17.793 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-24 15:10:17.794 - [Debug] [Auth] Attacker: 69.236.215.6 has so far made 1 attempts. Remaining: 5 attempts
2022-10-24 15:10:17.893 - [Debug] [Connection] Attacker closed the connection
2022-10-24 15:10:17.927 - [Debug] [Connection] Attacker connected: 69.236.215.6 | Client Identification: SSH-2.0-Go
2022-10-24 15:10:18.185 - [Debug] [Auth] Attacker 69.236.215.6 trying to authenticate with 'none'
2022-10-24 15:10:18.186 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-24 15:10:18.285 - [Debug] [Auth] Attacker 69.236.215.6 trying to authenticate with 'password'
2022-10-24 15:10:18.286 - [Debug] [Auto Access] Attacker: 69.236.215.6, Threshold: 1, Attempts: 1
2022-10-24 15:10:18.286 - [Debug] [Auto Access] Compromising the honeypot
2022-10-24 15:10:18.286 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-24 15:10:18.340 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-24 15:10:18.342 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-24 15:10:18.342 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.244
2022-10-24 15:10:18.364 - [Debug] [Auth] Attacker 31.165.5.155 trying to authenticate with 'none'
2022-10-24 15:10:18.364 - [Debug] [Auth] Attacker authentication error: No authentication method provided
